spacemacs/files.vim

```diff
@@ -1,50 +1,65 @@
 " Show bookmarks.
 " Create bookmark 0 with <C-S-0>, Create bookmark 1 with <C-S-1> and so on.
 " Go to bookmark 0 with <C-0>, go to bookmark 1 with <C-1> and so on.
+let g:WhichKeyDesc_Files_ShowBookmarks = "<leader>fb show-bookmarks"
 nnoremap <leader>fb    :action ShowBookmarks<CR>
 vnoremap <leader>fb    :action ShowBookmarks<CR>
 
+""" IDE Submenu ---------------------------------
+let g:WhichKeyDesc_Files_IDE = "<leader>fe +IDE"
+
 " Show settings
+let g:WhichKeyDesc_Files_IDE_Settings = "<leader>fed settings"
 nnoremap <leader>fed    :action ShowSettings<CR>
 vnoremap <leader>fed    <Esc>:action ShowSettings<CR>
 
 " Reload .ideavimrc
+let g:WhichKeyDesc_Files_IDE_ReloadIdeaVimRc = "<leader>feR reload-ideavimrc"
 nnoremap <leader>feR    :source ~/.ideavimrc<CR>
 vnoremap <leader>feR    <Esc>:source ~/.ideavimrc<CR>
+""" ---------------------------------------------
 
 " Find files
+let g:WhichKeyDesc_Files_GotoFileAlt = "<leader>fF goto-file"
 nnoremap <leader>fF    :action GotoFile<CR>
 vnoremap <leader>fF    :action GotoFile<CR>
+let g:WhichKeyDesc_Files_GotoFile = "<leader>ff goto-file"
 nnoremap <leader>ff    :action GotoFile<CR>
 vnoremap <leader>ff    :action GotoFile<CR>
 
-" Search in project
```

Comment:
Searching in project doesn't appear in my `develop` branch of Spacemacs, so I removed it from here.